src/cookie-utils.js

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+
```

Comment:
maybe I should just use js-cookie instead https://www.npmjs.com/package/js-cookie